toxav/msi.c

```diff
@@ -151,25 +152,72 @@ int msi_kill(MSISession *session, const Logger *log)
     pthread_mutex_unlock(session->mutex);
     pthread_mutex_destroy(session->mutex);
 
-    LOGGER_DEBUG(log, "Terminated session: %p", (void *)session);
+    LOGGER_API_DEBUG(tox, "Terminated session: %p", (void *)session);
     free(session);
     return 0;
 }
+
+/*
+ * return true if friend was offline and the call was canceled
+ */
+bool check_peer_offline_status(Tox *tox, MSISession *session, uint32_t friend_number)
+{
+    if (!tox) {
+        return false;
+    }
+
+    if (!session) {
+        return false;
+    }
+
+    TOX_ERR_FRIEND_QUERY f_con_query_error;
+    TOX_CONNECTION f_conn_status = tox_friend_get_connection_status(tox, friend_number, &f_con_query_error);
+
+    switch (f_conn_status) {
```

Comment:
This `switch` is an `if` in disguise. Could it be an `if` instead?

Reply:
done